lumen/filters/base.py

```diff
@@ -193,3 +196,25 @@ def panel(self):
         widget = self.widget.clone()
         self.widget.link(widget, value='value', bidirectional=True)
         return widget
+
+
+class ParamFilter(Filter):
+
+    parameter = param.ClassSelector(default=None, class_=(param.Parameter, str), doc="""
+        Reference to a parameter on an existing View.""")
```

Comment:
```suggestion
        Reference to a Parameter on an existing View.""")
```

(assuming you specifically mean param.Parameter)